lib/rules/web/ocm_console/cluster_detail_access_control_tab.xyaml

```diff
@@ -1,52 +1,41 @@
-check_users_tab:
-  element: &users_tab
+check_access_control_tab:
+  element: &access_control_tab
     selector:
-      xpath: //section[@id='clusterdetails-content']//div[@class='pf-c-tabs']/ul[@class='pf-c-tabs__list']//li/button[@class='pf-c-tabs__button' and text()='Users']
-click_users_tab:
+      xpath: //section[@id='clusterdetails-content']//div[@class='pf-c-tabs']/ul[@class='pf-c-tabs__list']//li/button[@class='pf-c-tabs__button' and text()='Access Control']
+click_access_control_tab:
   element:
-    <<: *users_tab
+    <<: *access_control_tab
     op: click
-check_add_user_button_in_empty_content_card:
-  element: &add_user_button
-    selector:
-      xpath: //section[@id='clusterdetails-content']//section[@id='usersTabContent']/div[@class='pf-c-empty-state pf-m-lg cluster-details-user-tab-contents']/button[@class='pf-c-button pf-m-primary' and text()='Add user']
-click_add_user_button:
-  element:
-    <<: *add_user_button
-    op: click
-check_empty_user_tab_content:
-  action: check_add_user_button_in_empty_content_card
+check_elements_on_user_card:
+  action: check_user_card_title
+  action: check_user_card_description
+  action: check_add_user_field
+check_user_card_title:
   element:
     selector:
-      xpath: .//parent::h5[@class="pf-c-title pf-m-lg" and text()='No users exist for this cluster']
-  element: &user_description_in_tab
-    selector:
-      xpath: .//parent::div[@class="pf-c-empty-state__body" and text()='You can add users to grant them access to the cluster. Users will be authenticated via your selected identity provider.']
-check_add_user_tab_content_without_users:
-  action: check_user_tab_content
-  action: check_add_user_field
-check_user_tab_content:
+      xpath: //section[@id='accessControlTabContent']/div[@class='cluster-details-user-tab-contents']/article/div[@class='pf-c-card__body']/h3[text()='Cluster Administrative Users']
+check_user_card_description:
   element:
     selector:
-      xpath: //div[@class='cluster-details-user-tab-contents']/p[text()='Grant permission to manage this cluster to users defined in your identity provider.']
+      xpath: //section[@id='accessControlTabContent']/div[@class='cluster-details-user-tab-contents']/article/div[@class='pf-c-card__body']/p[text()='Grant permission to manage this cluster to users defined in your identity provider.']/a[@href='https://docs.openshift.com/dedicated/4/administering_a_cluster/dedicated-admin-role.html' and text()='Learn more.']
```

Comment:
The xpath is too long to maintenance. You check too many elements in one xpath.. xpath is used for telling selenium the position of the elements, if the search condition is unique, the simplest xpath should be the best. For example, in this, you are checking two elements, one is the description `Grant permission ......`, another the link `Learn more`:
two elements are better, and `//p[text()='Grant permission to manage this cluster to users defined in your identity provider.']` is enough for the description. `Learn more` is similar too.